src/platform/main.c

```diff
@@ -75,7 +87,7 @@ int main(int argc, char *argv[])
     currentConsole = sdlConsole;
 #elif BROGUE_WEB
     currentConsole = webConsole;
-#else
+#elif BROGUE_CURSES
```

Comment:
Sure, no problem. I'll open another PR (for the `release` branch, right?)

There was no compile-time nor runtime errors. It only confused VS Code's C/C++ extension, because `cursesConsole` is declared behind `#ifdef BROGUE_CURSES` in `platform.h`.